test/cmdlineTests/model_checker_invariants_all/err

```diff
@@ -5,8 +5,8 @@ Warning: Return value of low-level calls not used.
   | 		^^^^^^^^^^^
 
 Info: Contract invariant(s) for model_checker_invariants_all/input.sol:test:
-((x <= 0) || true)
+(!(x >= 1) || true)
```

Comment:
Just curious, why this change? `(x <= 0)` and `!(x >= 1)` are equal, right?

Reply:
Yea, invariant and counterexample outputs are very nondeterministic, so the solver will choose an answer at its own discretion. As long as the results are correct you may get anything